js/models/BaseModel.js

```diff
@@ -116,8 +122,16 @@ export default class extends Model {
         const nestedInstance = this.attributes[nestedKey];
 
         if (nestedInstance) {
-          if (nestedData) nestedInstance.set(nestedData);
-          delete attrs[nestedKey];
+          try {
```

Comment:
hmm... I think it should work whether it's a getter or a function that returns an object, because the base model is accessing it via _.result() (`const nested = _.result(this, 'nested', []);`).

Yeah, if you find one way is not working, then please let me know... but Profile and the listing models seem to work and they each use different ways.

Anyhow... it's probably best, at least initially, to check in with us (or me for the nested model stuff) before tweaking some of the foundational classes too much. I understand with the timezone differences that might not be super easy.